SonarQube.Bootstrapper/BootstrapperSettings.cs

```diff
@@ -66,7 +67,7 @@ public class BootstrapperSettings : IBootstrapperSettings
         private readonly ILogger logger;
         private readonly string sonarQubeUrl;
         private readonly AnalysisPhase analysisPhase;
-        private readonly string childCmdLineArgs;
```

Comment:
Previously the bootstrapper concatenated the individual arguments into a string. It now skips the concatenation step and passes through the individual arguments.